lib/pos_blockchain.go

```diff
@@ -1800,7 +1800,7 @@ func (bc *Blockchain) GetUncommittedBlocks(tipHash *BlockHash) ([]*BlockNode, er
 
 // GetCommittedTipView builds a UtxoView to the committed tip.
 func (bc *Blockchain) GetCommittedTipView() (*UtxoView, error) {
```

Comment:
May as well update this too
```suggestion
func (bc *Blockchain) GetCommittedTipView() *UtxoView {
```

Reply:
Nvm, I see this is updated in the next PR. Please ignore